### ISSUE TITLE
Activity Log: missing actor name

### ISSUE THREAD

**Issue by MichaelArestad** [Automattic/wp-calypso#22166]
One of the events is missing the actor name.

![image](https://user-images.githubusercontent.com/1123119/35861317-854e2180-0b04-11e8-8fcb-fb9d7ab9e588.png)

Originally reported on p9rlnk-5z by @simison

**Comment by simison** [comment]
@gititon I believe this was already addressed/fixed?

**Comment by gititon** [comment]
Yes @simison , it should be. @MichaelArestad , can you please confirm whether this is an issue with the latest Jetpack?

**Comment by MichaelArestad** [comment]
I never had the issue. Just reported it. It looks like it's working for me.
